### PR TITLE
Added responsive layout to cancel popup

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**
Added responsive layout to cancel popup.

Breakpoints: 
-576x856;
-320x601;

### CODE REVIEW

File: src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@ $title-font: 'Montserrat';
 $main-font: 'Open Sans';
```
Comment (nabermi2020) [inline src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.scss:5]: image
Breakpoint 384

Comment (nabermi2020) [reply src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.scss:5]: image
Breakpoint 323

Comment (nabermi2020) [reply src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.scss:5]: image
Breakpoint 320

Comment (nabermi2020) [reply src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.scss:5]: Please, fix it according to the mockup.

Comment (RomanDeveloperAcc) [reply src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.scss:5]: Fixed

**Pull request merged**